cypress/integration/basic/group_modal.spec.js

```diff
@@ -0,0 +1,29 @@
+describe('Test that the group show modal can be opened', () => {
```

Comment:
In future work, write the describe() and it() descriptions such that they read together, e.g.:

describe('Group show modal')
it('can be opened')
it('has five tabs')

which would read: "[The] group show modal' ... can be opened ... has five tabs ..." etc.

I'm open to suggestions on this as we form a style guide.